script/ember.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/sh
+set -ue
+
+export FASTBOOT_DISABLED
+
+if [ "${USE_FASTBOOT:-0}" = '1' ]; then
```

Comment:
oh oops it needs to be `1` here? let me try that

Reply:
ember.sh is only used from npm run, but start-web.sh is doing the same check. Yes. It has to be 1